### PR TITLE
8: Dockerize the ASPSP simulator

### PR DESCRIPTION
**Pull request by matt-wills**
This PR introduces:

 - The jib-maven-plugin to push docker image to our gcr docker registry during maven build
 - A docker-compose file to start up the ASPSP simulator with MongoDB

Issue: SecureBankingAcceleratorToolkit/SecureBankingAcceleratorToolkit#8

### CODE REVIEW

**Review by jorgesanchezperez — APPROVED** [review]
Jib plugin seems cool, but It will work with github actions using docker credentials gcr?

**Comment by matt-wills** [comment]
> Jib plugin seems cool, but It will work with github actions using docker credentials gcr?

I hope so. There are quite a few authentication methods but we'll soon see!

https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#authentication-methods

File: pom.xml [hunk 1]
```diff
@@ -244,6 +245,37 @@
                     <artifactId>openapi-generator-maven-plugin</artifactId>
                     <version>${openapi-generator.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>com.google.cloud.tools</groupId>
+                    <artifactId>jib-maven-plugin</artifactId>
+                    <version>${jib-maven-plugin.version}</version>
+                    <configuration>
+                        <from>
+                            <image>openjdk:11-jre</image>
```
Comment (BohoCode) [inline pom.xml:254]: should be openjdk:14!

Comment (matt-wills) [reply pom.xml:254]: Oops! I'll change it in my next PR 👍🏻

**Pull request merged**